dotcom-rendering/src/server/render.article.web.tsx

```diff
@@ -223,6 +223,7 @@ window.twttr = (function(d, s, id) {
 		canonicalUrl,
 		renderingTarget: 'Web',
 		weAreHiring: !!article.config.switches.weAreHiring,
+		hasLiveBlogTopAd: article.config.hasLiveBlogTopAd,
```

Comment:
I'd suggest coercing to a boolean here, so that future travellers do not have to think about the `undefined` case:

```suggestion
		hasLiveBlogTopAd: !!article.config.hasLiveBlogTopAd,
```